src/main/java/io/github/javasemantic/commit/engine/factory/enums/ConventionalRuleEnum.java

```diff
@@ -13,10 +13,13 @@
 
   TYPE_RULE(TypeConventionalRule::new),
 
-  BREAKING_CHANGE_RULE(() -> new OptionalConventionalRule((commitComponent) ->
-      commitComponent.isExclamation() || (commitComponent.getBody() != null
-          && commitComponent.getBody().contains("BREAKING CHANGE"))
-  ));
+  BREAKING_BODY_RULE(() -> new OptionalConventionalRule((commitComponent) ->
+      commitComponent.getBody() != null
+          && commitComponent.getBody().stream().anyMatch(e -> e.contains("BREAKING CHANGE"))

```

Comment:
Maybe enums instead of having strings everywhere.

Reply:
Agreed